### DISCUSSION TITLE
Button nodes?

### DISCUSSION THREAD

**Discussion by mmontone** [neomacs-project/neomacs#90] [Q&A]
Is it possible to have button nodes?
I can create and insert a node with tag `button`, but I don't know how to bind its action, and make it readonly so that the button label cannot be edited.
I think an equivalent of Emacs' `insert-button` would be cool.

**Comment by kchanqvq** [comment]
I wrote an explanation of relevant mechanisms in the other discussion (`read-only` attribute and JavaScript IPC). It does make sense to wrap these in an `insert-button`-like API — maybe I'll add one in the next few days once I think I have a design!
